includes/mailpoet/class-sensei-mailpoet-repository.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * File containing the class Sensei_MailPoet_Repository.
+ *
+ * @since $$next-version$$
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Some Helper methods to fetch Sensei data: courses, groups and students.
+ */
+class Sensei_MailPoet_Repository {
+	/**
+	 * Get all enrolled students in a course or group.
+	 *
+	 * @since $$next-version$$
+	 * @param int    $id The post ID.
+	 * @param string $post_type The post type.
+	 *
+	 * @return array
+	 */
+	public static function get_students( $id, $post_type ) {
+		global $wpdb;
+		if ( 'group' === $post_type ) {
+			if ( class_exists( 'Sensei_Pro_Student_Groups\Repositories\Group_Student_Repository', true ) ) {
+				$group_student_repo = new Sensei_Pro_Student_Groups\Repositories\Group_Student_Repository( $wpdb );
+				$student_ids        = $group_student_repo->find_group_students( $id );
+			}
+		}
+		if ( 'course' === $post_type ) {
+			$students    = get_comments(
+				array(
+					'post_id' => $id,
+					'type'    => 'sensei_course_status',
+					'status'  => 'any',
+				)
+			);
+			$student_ids = array_map(
+				static function( $student ) {
+					return $student->user_id;
+				},
+				$students
+			);
+		}
+		if ( ! $student_ids ) {
+			return array();
+		}
+
+		$args = array(
+			'include' => $student_ids,
+			'fields'  => array( 'id', 'user_email', 'display_name', 'user_nicename' ),
+		);
+		return get_users( $args );
+	}
+
+	/**
+	 * Generates a Sensei LMS prefixed name for a MailPoet list.
+	 *
+	 * @since $$next-version$$
+	 * @param string $name The name of the course or group.
+	 * @param string $post_type The post type: course or group.
+	 *
+	 * @return string
+	 */
+	public static function get_list_name( $name, $post_type ) {
+		// translators: Placeholder is the post type singular name: Course or Group. The second placeholder is the Course or Group name.
+		return sprintf( __( 'Sensei LMS %1$s: %2$s', 'sensei-lms' ), get_post_type_object( $post_type )->labels->singular, $name );
```

Comment:
Looks like this one is the correct property name, sorry for the inaccuracy:
```suggestion
		return sprintf( __( 'Sensei LMS %1$s: %2$s', 'sensei-lms' ), get_post_type_object( $post_type )->labels->singular_name, $name );
```

Reply:
Looks like you are actually right!